demisto_sdk/commands/common/docker_util.py

```diff
@@ -0,0 +1,180 @@
+
+        Returns:
+            the file data in bytes
+        """
+        archive, stat = self.container.get_archive(f'{file_src}')
+        byte_file = io.BytesIO(initial_bytes=b''.join(archive))
+        byte_file.seek(0)
+        with tarfile.open(fileobj=byte_file) as tar_file:
+            io_obj = tar_file.extractfile(file_src.split(os.path.sep)[-1])
+            return io_obj.read() if io_obj else b''
+
+    def remove_container(self):
+        """
+        removing the container (e.g. docker rm)
+        """
+        if self._container_obj:
+            self._container_obj.remove()
+            self._container_obj = None
+
+    def remove_images(self):
+        """
+        removing the images created by the commits along the containers live (see the explanation in the exec comment)
+        """
+        self.remove_container()
+        if not (self._image_name == self._base_image_name):
+            DockerTools.remove_image(self._image_name)
+            self._image_name = self._base_image_name
+
+    def __del__(self):
+        self.remove_images()
+
+    @staticmethod
+    def _create_tar_info(name: str, file_bytes: bytes):
+        file_data = io.BytesIO(initial_bytes=file_bytes)
+        file_data.name = name
+        file_data.seek(0)
+        tar_info = tarfile.TarInfo(name=name)
+        tar_info.size = file_data.getbuffer().nbytes
+        return {'tarinfo': tar_info, 'fileobj': file_data}
+
+
+class DockerTools:
+    """
+    docker extra tools that will be usefully also as stand alone commands
+    """
+    @staticmethod
+    def remove_container(name_or_id: str, ignore_container_not_found: bool = True, force: bool = False):
+        """
+        Examples
+            docker rm MyTestContainer
+            can be replaced with
+            DockerTools.remove_container("MyTestContainer")
+            in your code
+        Args:
+            name_or_id: the name or id of the container
+            ignore_container_not_found: dont raise an exception in that the container is already not existing
+            force: like the -f in docker rm command
+
+        Raises:
+            docker.errors.NotFound in case that the container doesn't exist
+        """
+        try:
+            client.containers.get(name_or_id).remove(force=force)
+        except docker.errors.NotFound as not_found:
+            if not ignore_container_not_found:
+                raise not_found
+
+    @staticmethod
+    def remove_image(name_or_id: str, ignore_image_not_found: bool = True, force: bool = False):
+        """
+        Examples
+            docker rmi MyTestImage
+            can be replaced with
+            DockerTools.remove_image("MyTestImage")
+            in your code
+        Args:
+            name_or_id: the name or id of the image
+            ignore_image_not_found: dont raise an exception in that the image is already not existing
+            force: like the -f in docker rmi command
```

Comment:
```suggestion
            ignore_image_not_found: don't raise an exception in case the image already removed
            force: same as the -f option in the "docker rmi" command
```